src/ui/Templates/CoreHelpers.ts

```diff
@@ -569,7 +569,12 @@ TemplateHelpers.registerTemplateHelper(
 );
 
 TemplateHelpers.registerTemplateHelper('highlightStreamHTMLv2', (content: string, options: IHelperStreamHighlightOptions) => {
-  return executeHighlightStreamHTML(content, options);
+  const mergedOptions = {
```

Comment:
Rather than duplicating the merge here and on lines 559 - 566, plus other lines which are initializing two different parameters to the resolved values, could it be better to move the logic into `executeHighlightStreamHTML`? or separate function called inside that function? e.g.

```
const executeHighlightStreamHTML = (content: string, options: IHelperStreamHighlightOptions) => {
    const allOptions = populateMissingOptions(options);
    ...
}

const populateMissingOptions = (options: IHelperStreamHighlightOptions) => {
    const termsToHighlight = resolveTermsToHighlight();
    const phrasesToHighlight = resolvePhrasesToHighlight();

    return { termsToHighlight, phrasesToHighlight, ...options };
}
```